lib/theme.json

```diff
@@ -352,6 +352,28 @@
 							}
 						}
 					]
+				},
+				"grid": {
+					"name": "grid",
+					"slug": "grid",
+					"className": "is-layout-grid",
+					"displayMode": "grid",
+					"baseStyles": [
+						{
+							"selector": " > *",
+							"rules": {
```

Comment:
It seems there are more nuances about `margin/padding` when set in individual blocks inside the `grid`(example `heading` with margin inside `grid`). Can we contain them inside the grid column?

Reply:
Ah, I added that margin override due to [this comment](https://github.com/WordPress/gutenberg/pull/49018#discussion_r1134750872) pointing out that default block margins can interfere with the grid gap. It's the same we have for flex blocks; do you think this might be more of an issue with grid than with flex layouts?

There's also a whole discussion around margins set in global styles being overridden by layout rules, that we're trying to address in #47858.

Reply:
>do you think this might be more of an issue with grid than with flex layouts?

Probably the same :). Thanks for linking the discussions/issues.